pkg/justification/processor.go

```diff
@@ -163,8 +164,14 @@ func (p *Processor) createToken(ctx context.Context, now time.Time, req *jvspb.C
 	exp := now.Add(req.Ttl.AsDuration())
 	justs := req.Justifications
 
+	// Use audiences in the request if provided.
+	aud := []string{DefaultAudience}
```

Comment:
I might just be me, but this feels weird to alloc the aud that you might not use. Consider:

```go
aud := req.Audiences
if len(aud) == 0 {
  aud = []string{DefaultAudience}
}
```